README.md

```diff
@@ -95,6 +96,28 @@ Subsystem by running the following in PowerShell as an administrator
 Restart-Service LxssManager*
 ```
 
+### Local Development Login
+
+To log in to the system once locally deployed, simply enter a username in the designated field and click the "Login" button.
```

Comment:
I aim to avoid using language like 'simply' when writing user documentation.  Using words to describe an action as 'simply' implies that this should be easy for the user which is telling them how to feel about the feature. No one likes to get told how they feel or experience something.  It's a good habit to up with in documentation.


```suggestion
To log in to the system once locally deployed, enter a username in the designated field and click the "Login" button as seen in the figure below.
```